Jellyfin.Api/Controllers/UserController.cs

```diff
@@ -205,13 +205,20 @@ public async Task<ActionResult<AuthenticationResult>> AuthenticateUser(
     /// </summary>
     /// <param name="request">The <see cref="AuthenticateUserByName"/> request.</param>
     /// <response code="200">User authenticated.</response>
+    /// <response code="401">Invalid authorization cookie.</response>
     /// <returns>A <see cref="Task"/> containing an <see cref="AuthenticationRequest"/> with information about the new session.</returns>
     [HttpPost("AuthenticateByName")]
     [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
```

Comment:
401 is already added by https://github.com/jellyfin/jellyfin/blob/master/Jellyfin.Server/Filters/SecurityRequirementsOperationFilter.cs

Reply:
But it doesnt show up on the api docs https://api.jellyfin.org/#tag/User/operation/AuthenticateUserByName